scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/tasks/TasksListGrid.java

```diff
@@ -112,8 +112,9 @@ public TasksListGrid(TasksController controller, TasksColumnsFactory factory, St
     @Override
     public void build() {
         super.build();
+        this.setSelectionType(SelectionStyle.SINGLE);
         this.setSelectionProperty("isSelected");
-        this.sort(TasksColumnsFactory.ID_ATTR.getName(), SortDirection.ASCENDING);
+        this.sort(TasksCentricColumnsFactory.JOB_ID_ATTR.getName(), SortDirection.DESCENDING);
```

Comment:
Why do we change sort direction?

Reply:
Because we want to see on top the tasks belonging to the most recent jobs, which was not the case before and the visualization was messy.